frontend/src/components/partials/Footer/Footer.js

```diff
@@ -2,8 +2,6 @@ import React from "react"
 import "./footer.sass"
 export default function Footer(){
     return(
-        <footer className="w-100 bg-main-800">
-
-        </footer>
+        <footer className="w-100 bg-main-800"></footer>
```

Comment:
I think empty elements should be self closed
`<footer className="w-100 bg-main-800" />`